<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dashboard</class>
 <widget class="QDialog" name="dashboard">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="background_image">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>971</width>
     <height>711</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/dashboard/dashboard.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="icon_gestion_patient">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/dashboard/icon_gestion_patient.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_gestion_patient">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/dashboard/btn_gestion_patient1.png)3 10 15 10;
font: 24pt &quot;Bahnschrift&quot;;
</string>
   </property>
   <property name="text">
    <string>  Gestion des patients</string>
   </property>
  </widget>
  <widget class="QLabel" name="icon_gestion_login">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/login/img/login/logologin.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_gestion_login">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/dashboard/btn_gestion_patient1.png)3 10 15 10;
font: 24pt &quot;Bahnschrift&quot;;
</string>
   </property>
   <property name="text">
    <string>  Gestion de Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-30</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Connect√© en tant que :</string>
   </property>
  </widget>
  <widget class="QLabel" name="login_active">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>-30</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>background_image</zorder>
  <zorder>btn_gestion_patient</zorder>
  <zorder>icon_gestion_patient</zorder>
  <zorder>btn_gestion_login</zorder>
  <zorder>icon_gestion_login</zorder>
  <zorder>label</zorder>
  <zorder>login_active</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
